### PR TITLE
[G2M] AWS Region variablised, FLUX CLI added, Updated Readme

### PR DESCRIPTION
**Pull request by rishabhgargg**

### CODE REVIEW

File: single-liner/new.sh [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
-aws configure set region us-east-1
+aws configure set region ${AWS_REGION}
 aws configure set aws_access_key_id ${AWS_ACCESS_KEY_ID}
 aws configure set aws_secret_access_key ${AWS_SECRET_ACCESS_KEY}
 aws eks update-kubeconfig --region us-east-1 --name ${NAME_OF_CLUSTER}
+
+#Extra Utilities. 
+#If value of the corresponding env variable is not set, it will not be installed.
+############ FLUX CLI ##############
+if [[ -n "$FLUX_VERSION" ]]; then
```
Comment (ashishjullia) [inline single-liner/new.sh:12]: @rishabhgargg 
~How will `$FLUX_VERSION` will get env value and from where?~
Also, why no `${ }` for `$FLUX_VERSION` as it is the case with other available env vars in the same file.

File: single-liner/new.sh [hunk 2]
```diff
@@ -1,8 +1,16 @@
 #!/bin/bash
 tfenv install ${TF_VERSION} 
 tfenv use ${TF_VERSION}  
-aws configure set region us-east-1
+aws configure set region ${AWS_REGION}
 aws configure set aws_access_key_id ${AWS_ACCESS_KEY_ID}
 aws configure set aws_secret_access_key ${AWS_SECRET_ACCESS_KEY}
 aws eks update-kubeconfig --region us-east-1 --name ${NAME_OF_CLUSTER}
```
Comment (ashishjullia) [inline single-liner/new.sh]: @rishabhgargg I think we can leverage the same env approach for `${AWS_REGION}` here?

File: single-liner/new.sh (hunk 1 above)
Comment (rishabhgargg) [reply single-liner/new.sh:12]: $FLUX_VERSION will get it's value from .env file (Updated Readme file in same PR for reference).
For your second query, the FLUX_VERSION variable is used as env variable inside the install.sh script that is being pulled in line 13.

Comment (rishabhgargg) [reply single-liner/new.sh:12]: Removing existing `{ }` for consistency.

**Review by ashishjullia — APPROVED** [review]

**Pull request merged**